airbyte-integrations/connectors/destination-oracle/src/test-integration/java/io/airbyte/integrations/destination/oracle/OracleIntegrationTest.java

```diff
@@ -172,21 +152,18 @@ private static Database getDatabase(JsonNode config) {
 
   @Override
   protected void setup(TestDestinationEnv testEnv) throws SQLException {
-    config = getConfig(db);
-
-    final Database database = getDatabase(config);
-    database.query(ctx -> {
-      ctx.execute("alter database default tablespace users");
-      return null;
-    });
```

Comment:
I removed this because requires a very strong privilege to execute it. I'm running the AWS instance where I can change the permission of my admin user.